examples/atlas-search.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * This example demonstrates how to create an Atlas Search index and perform a search query.
+ * It requires a MongoDB Atlas M10+ cluster with Sample Dataset loaded.
+ *
+ * Use the MONGODB_URI environment variable to specify the connection string from the Atlas UI.
+ */
+
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use Closure;
+use MongoDB\Client;
+use RuntimeException;
+
+use function define;
+use function getenv;
+use function hrtime;
+use function iterator_to_array;
+use function sleep;
+use function str_contains;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI');
```

Comment:
```suggestion
$uri = getenv('MONGODB_URI') ?? null;
```

This may address the PossiblyFalseArgument error below. Alternatively, default to an empty string and then you can change the following line to `if (! str_contains(...))` and remove the `! $uri` condition.